src/test/java/com/wikia/webdriver/testcases/adstests/TestATF.java

```diff
@@ -0,0 +1,79 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.contentpatterns.AdsContent;
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.dataprovider.ads.AdsDataProvider;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.common.templates.TemplateDontLogout;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsBaseObject;
+
+import org.openqa.selenium.Dimension;
+import org.testng.annotations.Test;
+
+
+/**
+ * @author drets
+ * @ownership AdEng
+ */
+public class TestATF extends TemplateDontLogout {
+
+  private final static Dimension PAGE_SIZE = new Dimension(1366, 768);
+
+  @Test(
+      dataProviderClass = AdsDataProvider.class,
+      dataProvider = "delayBtf",
+      groups = {"TestATF", "Ads"}
+  )
+  public void TestATFDelayBTF(String wikiName, String article, int delaySec, boolean isWgVarOn)
+      throws InterruptedException {
+    PageObjectLogging.log("wgAdDriverDelayBelowTheFold", String.valueOf(isWgVarOn), true);
+
+    String testedPage = urlBuilder.getUrlForPath(wikiName, article);
+    AdsBaseObject adsBaseObject = new AdsBaseObject(driver, testedPage, PAGE_SIZE);
+    adsBaseObject.waitPageLoaded();
+
+    Assertion.assertTrue(adsBaseObject.checkIfSlotOnPageLoaded(AdsContent.MEDREC));
+    Assertion.assertTrue(adsBaseObject.checkIfSlotOnPageLoaded(AdsContent.TOP_LB));
+    Assertion.assertTrue(adsBaseObject.checkIfSlotOnPageLoaded(AdsContent.INVISIBLE_SKIN));
+
+    if (isWgVarOn) {
+      Assertion.assertFalse(adsBaseObject.checkIfSlotOnPageLoaded(AdsContent.PREFOOTER_LEFT));
+      Assertion.assertFalse(adsBaseObject.checkIfSlotOnPageLoaded(AdsContent.PREFOOTER_RIGHT));
+      Assertion.assertFalse(adsBaseObject.checkIfSlotOnPageLoaded(AdsContent.LEFT_SKYSCRAPPER_3));
+      Assertion.assertFalse(adsBaseObject.checkIfSlotOnPageLoaded(AdsContent.FLOATING_MEDREC));
+      Thread.sleep(1000 * delaySec);
```

Comment:
The scenario is following:
(if wgVar is set to true for wiki)
* check that slots are not shown
* wait 20 sec
* check that slots are appeared

I configured the ads in this page in such a way that after 20 sec it should appear.
I don't see how can I use explicit wait here. I am fine with `Thread.sleep` because delay is part of functionality.